scripts/setup_dev_environment.sh

```diff
@@ -7,6 +7,8 @@
 # - set up a test oauth2 rp (https://kanidm.com)
 # - prompt to reset testuser's creds online
 
+set -e
```

Comment:
this seems unrelated, and idk what it'd do

Reply:
It's making the script fail on any errors, which is important.